src/Tabs/TabsWrapper.php

```diff
@@ -206,7 +206,7 @@ function twAddFrameTab(tabsid, label, url) {
       temp = panelId;
       return panelId;
     } else {
-      asyncAlertMsg('Fee Sheet tab is already opened', 3000, type = 'warning', size = '') ;
+      asyncAlertMsg($message, 3000, 'warning','') ;
```

Comment:
always the best way. I hardly ever translate inside a function call and my first example showed defining message then call function with it. Because i'm lazy, I passed on a bad habit in last review. Sorry.